man/g.plot5.Rd

```diff
@@ -31,12 +40,18 @@
   \item{overwrite}{
     See \link{g.shell.GGIR}
   }
-  \item{metric}{
-    Which one of the metrics do you want to consider to describe behaviour. The metric of interest need to be calculated in M (see \link{g.part1})
-  }
   \item{desiredtz}{
     See \link{g.getmeta}
   }
+  \item{threshold.lig}{
+    See \link{g.shell.GGIR}
```

Comment:
replace all three of these by:

`See \link{g.part5}`

because that is where they are documented

Reply:
oh right, my error